packages/order-watcher/src/order_watcher/order_watcher.ts

```diff
@@ -57,6 +57,10 @@ interface OrderStateByOrderHash {
     [orderHash: string]: OrderState;
 }
 
+export interface Stats {
+    orderCount: number;
+}
```

Comment:
Please move this to `order-watcher/src/types.ts` since we export it.

Then import it in this file, and export from `index.ts`.

Reply:
Let's make sure to export this type from `index.ts` since it's part of the public interface of this package.